boards/nxp/mimxrt1170_evk/board.cmake

```diff
@@ -6,10 +6,7 @@
 
 if(CONFIG_SOC_MIMXRT1176_CM7 OR CONFIG_SECOND_CORE_MCUX)
  board_runner_args(pyocd "--target=mimxrt1170_cm7")
- board_runner_args(jlink "--device=MIMXRT1176xxxA_M7" "--reset-after-load")
- # ITCM is not defined in RT1170's LinkServer device file
- board_runner_args(linkserver "--override=/device/memory/-=\{\"location\":\"0x00000000\",\
-                               \"size\":\"0x00040000\",\"type\":\"RAM\"\}")
```

Comment:
the comment here makes me think once again that the device memory map linkserver issue is not a zephyr problem

Reply:
This code is actually wrong, and Linkserver has the correct mapping for this address space. Specifically, we gave the wrong size when this line was created but again no longer needed since Linkserver has the correct address mapping.